CHANGELOG.unreleased.md

```diff
@@ -6,7 +6,10 @@
 
 ### Bug fixes
 
+- [SDN Controller] Monitor host lifecycle with their metrics `live` field instead of their `enabled` one. (PR [#4314](https://github.com/vatesfr/xen-orchestra/pull/4314))
```

Comment:
IMHO, I don't think that clients need this much of details. 
Maybe `Fix host life cycle monitor`